### PR TITLE
Simulate crawler behavior we've been seeing

### PR DESCRIPTION
**Pull request by coverbeck**
Tied to https://ucsc-cgl.atlassian.net/browse/SEAB-4514. Please see details on this PR [here](https://ucsc-cgl.atlassian.net/browse/SEAB-4514?focusedCommentId=42418).

### CODE REVIEW

File: src/test/scala/io/dockstore/Requests.scala [hunk 1]
```diff
@@ -103,6 +103,16 @@ object Requests {
       if (token != null && !token.isEmpty) builder.headers(authHeader(token)) else builder
     }
 
+    /**
+     * Requires session attributes encodedTrsId, version, and descriptorType be set
+     * @return
+     */
+
+    def getPlainDescriptor() = {
+      val path = "/api/api/ga4gh/v2/tools/#{encodedTrsId}/versions/#{version}/PLAIN_#{descriptorType}/descriptor"
```
Comment (denis-yuen) [inline src/test/scala/io/dockstore/Requests.scala:112]: don't know if it matters, but this is testing trs v2 beta, not trs v2 final

**Review by denis-yuen — APPROVED** [review]

File: src/test/scala/io/dockstore/Requests.scala (hunk 1 above)
Comment (coverbeck) [reply src/test/scala/io/dockstore/Requests.scala:112]: Grr, that wasn't the intent.

File: src/test/scala/io/dockstore/Requests.scala [hunk 2]
```diff
@@ -103,6 +103,16 @@ object Requests {
...
+      http(s"Get ${path}").get(path)
```
[the remark this answers is not in the document]
Comment (coverbeck) [reply src/test/scala/io/dockstore/Requests.scala]: The beauty of Scala :) At least some parts of it.

**Review by kathy-t — APPROVED** [review]

File: src/test/scala/io/dockstore/Requests.scala (hunk 1 above)
Comment (denis-yuen) [reply src/test/scala/io/dockstore/Requests.scala]: FWIW, Terra is still v2 beta I thought, so maybe more realistic. 
v2 final is what we care about for the others like DNAstack, could also load test a mix of the two
either way is probably ok in the broader strokes

Comment (coverbeck) [reply src/test/scala/io/dockstore/Requests.scala]: I'm pretty sure Terra is using v1.

We should move the other tests to v2 final, but I wanted to limit the scope of this PR for the new test.

**Pull request merged**